app/helpers/ads_helper.rb

```diff
@@ -8,29 +8,44 @@ def city_and_zip(ad)
   end
 
   def filtered?
-    params[:service].present? || params[:city].present?
+    params[:category].present? || params[:city].present?
+  end
+
+  def categories_for_select
+    %w[
```

Comment:
Q: Can you reuse Ad::CATEGORIES here?

Reply:
I redefined them here because of the order :)